CONTRIBUTING.md

```diff
@@ -144,9 +144,9 @@ Please take a moment to check whether an issue already exists. If it does, it ne
 
 ### Pull Requests always welcome
 
-We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. Not sure if that typo is worth a pull request? Do it! We will appreciate it.
+We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. However, if you are not sure if that typo is worth a pull request? Do it! We will appreciate it.
 
-If you don't get a response to a pull request within a few days, feel free to mention it in the #ipfs IRC channel. We may be able to get to it faster.
+If you don't get a response to a pull request within a few days, feel free to ping the repository maintainer, captain or last person to merge something in that repository.
```

Comment:
I see people posting on IRC and not getting any help because we don't look so much into it. Finding who to ping on IRC is equivalent to finding who to ping directly on the issue and does not require getting out of github and figuring out what is IRC and what our channels are.

Reply:
Fair enough. But I also get quite a few direct pings on IRC for things I've been ignoring on github. It's nice having a second channel.

Maybe both?

Reply:
ok, how does it look now?

Reply:
:+1: